specificatie/openapi.yaml

```diff
@@ -683,14 +971,6 @@ components:
       - $ref: "#/components/schemas/Adres"
       - description: "Het adres is de benaming van een locatie bestaande uit straatnaam, huisnummer, (mogelijk met huisletter, huisnummertoevoeging, postcode) en woonplaatsnaam. Dit is de vereenvoudigde samenstelling van de officiële naamgeving gebaseerd op woonplaats, openbare ruimte en nummeraanduiding."
         properties:
-          adresregel1:
```

Comment:
aangezien de adresregels in deze sprint zijn gebouwd, moeten ze niet meer met dit pull request worden verwijderd

Reply:
ik voeg ze toe.